### ISSUE TITLE
Admin\Contacts - Internal User can view a contact

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1698]
#### Description:
As an internal user, I can see the contact details for each contact 

#### Acceptance Criteria:

Given I am on the [Internal contact grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7854-90&t=Rp9uqAoG8OacaBH6-4)
When I click view details on a contact
Then I am brought to the contact details page for that contact
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=28402-1256&t=Rp9uqAoG8OacaBH6-4)

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by patriciarussellCAS** [comment]
Hey @zoeyli-46 could we update the wireframe link so I can see what this is for? 
Trying to figure out if this is an MVP requirement or not. Thanks!

**Comment by zoeyli-46** [comment]
@patriciarussellCAS wireframe link updated! For context, its just so the internal user can see their address, job title, contact info, and where they are assigned. If we needed to scope down for MVP, then perhaps the most important thing is just the contact email/phone number + place assigned and have that displayed on the grid itself? and the remaining details could be shown in metabase?

**Comment by Sepehr-Sobhani** [comment]
All internal users can now view the contacts grid and the contact details page. Can we consider this ticket resolved?

**Comment by patriciarussellCAS** [comment]
Agreed. I'll close this.
